components/bl-drawer-component/README.md

```diff
@@ -0,0 +1,87 @@
+# Drawer
+
+Drawer is the component for Backendless [UI-Builder](https://backendless.com/developers/#ui-builder). It allows the user
+to open a panel that is overlaid on top of a page and slides in from the edge of the screen.
+
+<p align="center">
+  <img src="./thumbnail.png" alt="main thumbnail" width="780"/>
+</p>
+
+## Properties
+
+| Property                      | Type                                                  | Default value | Logic              | Data Binding | UI Setting | Description                                                      |
+|-------------------------------|-------------------------------------------------------|---------------|--------------------|--------------|------------|------------------------------------------------------------------|
+| Open Drawer Button Visibility | *Checkbox*                                            | `true`        |                    | NO           | YES        | Enables showing the Open Drawer button.                          |
+| Open Drawer Button Label      | *Text*                                                | "Open Drawer" |                    | NO           | YES        | Controls the label of the Open Drawer button.                    |
```

Comment:
Could we make it shorter?

Control Visibility  and Control Label or something like that.